virttest/utils_misc.py

```diff
@@ -456,41 +414,32 @@ def set_log_file_dir(directory):
 
     :param dir: Directory for log files.
     """
-    global _log_file_dir
-    _log_file_dir = directory
+    logging.warning("Calling log functions from `utils_misc` is deprecated, "
+                    "please use `utils_logfile` for the purpose")
+    utils_logfile.set_log_file_dir(directory)
 
 
 def get_log_file_dir():
     """
     get the base directory for log files created by log_line().
 
     """
-    global _log_file_dir
-    return _log_file_dir
+    logging.warning("Calling log functions from `utils_misc` is deprecated, "
+                    "please use `utils_logfile` for the purpose")
+    return utils_logfile.get_log_file_dir()
 
 
 def get_log_filename(filename):
     """return full path of log file name"""
-    return get_path(_log_file_dir, filename)
+    logging.warning("Calling log functions from `utils_misc` is deprecated, "
+                    "please use `utils_logfile` for the purpose")
+    return utils_logfile.get_log_filename(filename)
 
 
 def close_log_file(filename):
-    global _open_log_files, _log_file_dir, _log_lock
-    remove = []
-    if not _acquire_lock(_log_lock):
-        raise LogLockError("Could not acquire exclusive lock to access"
-                           " _open_log_files")
-    try:
-        for k in _open_log_files:
-            if os.path.basename(k) == filename:
-                f = _open_log_files[k]
-                f.close()
-                remove.append(k)
-        if remove:
-            for key_to_remove in remove:
-                _open_log_files.pop(key_to_remove)
-    finally:
-        _log_lock.release()
+    logging.warning("Calling log functions from `utils_misc` is deprecated, "
+                    "please use `utils_logfile` for the purpose")
+    return utils_logfile.close_log_file()
```

Comment:
Here missed the `filename` to be passed to the underlying.

Reply:
Good catch!